server.py

```diff
@@ -1,14 +1,22 @@
 import socket as s
+import os
+import mimetypes
 
 
-def response_ok(reason_phrase):
-    return "HTTP/1.1 200" + reason_phrase + "\r\n"
-    "Content-Type: text/html; charset=utf-8\r\n"
+ROOT_DIRECTORY = os.path.join(__file__, b'webroot/')
+
+
+def response_ok(uri):
+    length = str(len(resolve_uri(uri)[0]))
+    response = b"HTTP/1.1 200 OK\r\nContent-Type: {}; charset=utf-8\
+    \r\nContent-Length: {}\r\n\r\n {}".format(resolve_uri(uri)[1],
+                                              length, resolve_uri(uri)[0])
+    return response
 
 
 def response_error(error_code, reason_phrase):
-    response = "HTTP/1.1 " + error_code + " " + reason_phrase + "\r\n"
-    "Content-Type: text/html; charset=utf-8\r\n"
+    response = b"HTTP/1.1 {} {}\r\nContent_Type: text/html;\
+    charset=utf-8\r\n".format(error_code, reason_phrase)
```

Comment:
gah.  I hate the line-continuation character.  Is there some other way to do this?  Perhaps make the template string on one line, and format it on the next?